deploy/charts/cert-manager/cainjector/templates/deployment.yaml

```diff
@@ -4,25 +4,28 @@ metadata:
   name: {{ include "cainjector.fullname" . }}
   namespace: {{ .Release.Namespace | quote }}
   labels:
-    app: {{ include "cainjector.name" . }}
-    chart: {{ include "cainjector.chart" . }}
-    release: {{ .Release.Name }}
-    heritage: {{ .Release.Service }}
+    app.kubernetes.io/name: {{ include "cainjector.name" . }}
+    app.kubernetes.io/instance:  {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    helm.sh/chart: {{ include "cainjector.chart" . }}
 spec:
   replicas: {{ .Values.replicaCount }}
   selector:
     matchLabels:
-      app: {{ include "cainjector.name" . }}
-      release: {{ .Release.Name }}
+      app.kubernetes.io/name: {{ include "cainjector.name" . }}
+      app.kubernetes.io/instance:  {{ .Release.Name }}
+      app.kubernetes.io/managed-by: {{ .Release.Service }}
```

Comment:
How will this affect existing deployments that already use the old labels? IIRC, this can leave orphaned pods lying around? I may be wrong here 😄 

If so, perhaps it'll be best to make this PR *additive* instead of replacing the old labels altogether?

Reply:
done, added the app label back